model/event/event_finder.go

```diff
@@ -8,6 +8,31 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var embeddedResourceTypeKey = DataKey + "." + resourceTypeKey
```

Comment:
can we wrap this in a function? I'd rather avoid package mutables?

I'd also be inclined to just use bsonutil.GetDottedKeyName()

Reply:
done